press/hooks.py

```diff
@@ -355,3 +355,61 @@
 page_renderer = ["press.metrics.MetricsRenderer"]
 
 export_python_type_annotations = True
+
+
+# These are used for some business logic, they should be manually evicted.
+__persistent_cache_keys = [
+	"agent-jobs",
+	"monitor-transactions",
+	"google_oauth_flow*",
+	"fc_oauth_state*",
+	"one_time_login_key*",
+	"press-auth-logs",
+]
+
+# `frappe.rename_doc` erases all caches, this hook preserves some of them.
+# Note:
+# - These are only "most used" cache keys. This lessens the impact of renames but doesn't eliminate them.
+# - Adding more keys here will slow down `frappe.clear_cache` but it's "rare" enough.
+# - This also means that other "valid" frappe.clear_cache() usage won't clear these keys!
+# - Use frappe.cache.flushall() instead.
+persistent_cache_keys = [
+	*__persistent_cache_keys,
+	"agent_job_step_output",
```

Comment:
This one isn't evicted anywhere btw, it can get huge but we have tons of memory so :shrug: